hs_err_pid71772.log

```diff
@@ -0,0 +1,1349 @@
+#
```

Comment:
Remove this file please

Reply:
Beware this also contains your real name.

Reply:
Yoink, thanks for the heads up

Reply:
Deleted in [95168f9](https://github.com/Sollace/Void-Fog/pull/13/commits/95168f96f06bcdbbb1a3950638afe597f7af1a3f)